UNRELEASED.md

```diff
@@ -10,6 +10,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Updated the type of the `title` prop in `ChoiceList` from `string` to `ReactNode` ([#2355](https://github.com/Shopify/polaris-react/pull/2355))
+
 ### Bug fixes
 
 - Fixed an accessibility issue with `TextField` `multiline` where `aria-multiline` would be set to an invalid type `number` ([#2351](https://github.com/Shopify/polaris-react/pull/2351))
```

Comment:
@BPScott @LauraAubin I assume these have been re-introduced by the latest revert - should I removed them in this PR?

Reply:
Unsure of the top of my head - rebase atop latest master and see what ends up being in this file

Reply:
That's what I did :o 

As you can see the latest version of the [file](https://github.com/Shopify/polaris-react/blob/master/UNRELEASED.md), there are references to stuff that have been merged and shipped already - hence my confusion 

From what I understand from the [guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)
> The changelog is prepared manually immediately before a release, by moving changelog entries from UNRELEASED.md to CHANGELOG.md, under a new heading for the version number.

We have to clear the file manually?

Reply:
Those entries must have not been cleared from the unreleased file with the latest release. Since they are referenced in the changelog, and I don't see a revert, I think it's fine to remove those entries from the unreleased doc.

Reply:
I was referring to https://github.com/Shopify/polaris-react/pull/2377, anyways I can clear the doc 👍